ci-operator/config/stolostron/acmqe-grc-test/stolostron-acmqe-grc-test-release-2.7.yaml

```diff
@@ -0,0 +1,20 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+images:
+- dockerfile_path: Dockerfile.interop
```

Comment:
it looks like the Dockerfile in this repo is `Dockerfile` not `Dockerfile.interop`

https://github.com/stolostron/acmqe-grc-test/tree/release-2.7

Reply:
oh I see, I will ask the GRC team to merge the code and then we will be able to see the Dockerfile.interop

Reply:
We could just use Dockerfile here instead, that's up to you though.

Reply:
better to use the interop one for OPP tests, not sure if the other one is tested

Reply:
/ascerra The PR is merged and the Dockerfile.interop is now in the release-2.7 branch